### ISSUE TITLE
Export "Extended Colors" from MTB Web

### ISSUE THREAD

**Issue by beqaMeqvabishvili** [material-foundation/material-theme-builder#53]
Currently Material Theme Builder Web has an option to extend color scheme, but extended colors aren't included in CSS or DSP exports (these are the only ones I've tried).

I'd prefer to have those available and not have to copy and paste variables from the browser, as each extended color generates 20+ color values

![image](https://user-images.githubusercontent.com/22842333/161860588-c8b90dee-117d-4120-82f6-3ad7c3861b2a.png)

**Comment by jwill** [comment]
Extended color is exported to some extent on Android Views, Compose, and Flutter.

Adjusting title of issue.

**Comment by jiblett1000** [comment]
I agree. This functionality would be very useful. The json output and perhaps css would be of most use to myself.

**Comment by danielduckworth** [comment]
+1 from me

**Comment by iurycarlos** [comment]
would be awesome!

**Comment by benjamincox** [comment]
@margeeta @rodydavis This also fails to export the generated extended color styles in the JSON. 

<img width="1394" alt="Screenshot 2023-07-29 at 3 37 59 PM" src="https://github.com/material-foundation/material-theme-builder/assets/137004/f7ed6471-d4c3-47c1-9db2-66eb2838cee9">
<img width="604" alt="Screenshot 2023-07-29 at 3 38 06 PM" src="https://github.com/material-foundation/material-theme-builder/assets/137004/fe55728e-9b99-4050-aa17-742cbdea49f7">
